webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/Formatters/CellFormatterWrapper.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2021 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import { useContext, useState } from 'react';
+import type { FormatterProps } from 'react-data-grid';
+import styled, { css } from 'reshadow';
+
+import { useObjectRef } from '@cloudbeaver/core-blocks';
+import type { IDataPresentationActions, IResultSetElementKey } from '@cloudbeaver/plugin-data-viewer';
+
+import { EditingContext } from '../../Editing/EditingContext';
+import { CellContext } from '../CellRenderer/CellContext';
+import { DataGridContext } from '../DataGridContext';
+import { TableDataContext } from '../TableDataContext';
+import { CellFormatterFactory } from './CellFormatterFactory';
+import { CellMenu } from './Menu/CellMenu';
+
+interface Props extends FormatterProps {
+  className?: string;
+}
+
+const styles = css`
+  formatter-wrapper {
+    flex: 1;
+    display: flex;
+  }
+  formatter-container {
+    flex: 1;
+  }
+`;
+
+export const CellFormatterWrapper: React.FC<Props> = observer(function CellFormatterWrapper({ className, ...rest }) {
+  const { column, rowIdx, isCellSelected } = rest;
```

Comment:
remove, `column` used once, `rowIdx` twice and `isCellSelected` also used once, use `rest.column` instead